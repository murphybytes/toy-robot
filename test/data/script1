PLACE 0,0,EAST
REPORT
ZIP
MOVE
REPORT
