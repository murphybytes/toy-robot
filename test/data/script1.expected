EAST (x,y) (0, 0)
EAST (x,y) (1, 0)
